client/app/services/dashboard.js

```diff
@@ -171,6 +189,7 @@ const DashboardService = {
   favorites: params => axios.get("api/dashboards/favorites", { params }).then(transformResponse),
   favorite: ({ id }) => axios.post(`api/dashboards/${id}/favorite`),
   unfavorite: ({ id }) => axios.delete(`api/dashboards/${id}/favorite`),
+  recentDashboards: params => axios.get("api/dashboards", { params }).then(transformRecentResponse)
```

Comment:
How about we have the same method name as in the Query, so we can have shared code? (it's something similar to what we do with the `favorite` ones)

Reply:
I got your idea, this would be great. But i cant figure out an meaningful name for both, i think something like "recents" would not say too much about the method, given that we also have this other "recent" feature.